resources/css/style.css

```diff
@@ -1,4 +1,4 @@
-.all-filters-option {
+.all-filters-option {  /* Ignore CSSLintBear */
```

Comment:
I tried to alter positions of `-webkit*` css styles, but it wasn't affecting the error. It was still getting raised.